app/src/main/java/com/futurice/freesound/feature/search/SearchActivity.java

```diff
@@ -102,6 +102,23 @@ public void bind(@NonNull final CompositeDisposable d) {
                                  .subscribe(SearchActivity.this::handleErrorState,
                                             e -> e(e, "Error receiving Errors")));
         }
+
+        private void subscribeToSearchView(@NonNull final SearchView view,
```

Comment:
It was a suggestion by the IDE that being flagged as a warning. I thought I'd clean it up while I was there (it's just moving the method).